api/apis/package_handler.go

```diff
@@ -183,7 +183,7 @@ func (h PackageHandler) packageCreateHandler(authInfo authorization.Info, w http
 	if err != nil {
 		switch err.(type) {
 		case repositories.ForbiddenError:
-			h.logger.Error(err, "Not authorized to create packages", "App Name", payload.Relationships.App)
+			h.logger.Info("Not authorized to create packages", "App Name", payload.Relationships.App, "error", err)
```

Comment:
Not truly related to this story, but an auth failure should not be logged as an error. That causes a stack trace in the logs implying an app error, when this is just a normal runtime failure.